code/modules/crew_objectives/cargo_objectives.dm

```diff
@@ -0,0 +1,81 @@
+/*				CARGO OBJECTIVES				*/
+
+/datum/objective/crew/petsplosion
+	explanation_text = "Ensure there are at least (If you see this, yell on GitHub) pets on the station by the end of the shift. Interpret this as you wish."
```

Comment:
So a lot of these objectives have those `yell on github` lines, but seemingly they're normal, properly coded objectives? Why are those here?

Reply:
Indeed.

Reply:
The reason I don't have a default generic error message for all of them is so we can easily tell which objective it is that's breaking.